src/Compilers/VisualBasic/Portable/Parser/Parser.vb

```diff
@@ -6171,15 +6171,17 @@ checkNullable:
             If feature = Feature.InterpolatedStrings Then
                 ' Bug: It is too late in the release cycle to update localized strings.  As a short term measure we will output 
```

Comment:
Filed https://github.com/dotnet/roslyn/issues/17807
I'll fix asap, but in a separate PR